pkgs/applications/emulators/cemu-wiiu/default.nix

```diff
@@ -0,0 +1,134 @@
+{ lib, stdenv, fetchFromGitHub
+, addOpenGLRunpath
+, wrapGAppsHook
+, cmake
+, glslang
+, nasm
+, pkg-config
+
+, SDL2
+, boost
+, cubeb
+, curl
+, fmt_9
+, glm
+, gtk3
+, imgui
+, libpng
+, libzip
+, libXrender
+, pugixml
+, rapidjson
+, vulkan-headers
+, wxGTK32
+, zstd
+
+, vulkan-loader
+}:
+
+let
+  version = "unstable-2022-10-28";
+  rev = "e0aaf631c460cd893c3ec682a237891f2d1baa50";
```

Comment:
Maybe we should use tagged release instead? https://github.com/cemu-project/Cemu/releases/tag/v2.0-10

Reply:
+1 on the tagged releases, especially since the build improvements were implemented and released in a tagged version the raw git repo no longer needs to be used.